patterns/README.md

```diff
@@ -896,3 +896,66 @@ of the following examples set a unique constraint on field `a`:
 ### Implementations
 
 None known.
+
+# Files in Zip (or other compressed files)
```

Comment:
I think we have wrong indentation level here - we need main headings to be `##` i think. Could you tweak this and then change rest of headings one level down.

Reply:
Also can we get a blank line after headings?